src/js/components/form-elements/FieldArrayComponent.jsx

```diff
@@ -38,24 +38,26 @@ class FieldArrayComponent extends Component {
         <div className="text-center border">
           <div className="d-flex flex-row border-bottom font-weight-bold py-1">
             {_.map(fieldsConfig.fields, (config, name) => (
-              <Tooltip
-                html={(config.label && <div>{this.props.translate(config.label)}</div>)}
-                theme="transparent"
-                arrow="true"
-                delay="150"
-                style={{
-                  flex: config.fixedWidth ? `0 1 ${config.fixedWidth}` : `${config.flexWidth || '12'} 1 0`,
-                  minWidth: 0,
-                }}
-                duration="250"
-                hideDelay="50"
+              <div style={{
+                flex: config.fixedWidth ? `0 1 ${config.fixedWidth}` : `${config.flexWidth || '12'} 1 0`,
```

Comment:
This line does not make sense. If you are checking if config.fixedWidth exists then why in case if not you have ${config.flexWidth || '12'?

Reply:
Ups my mistake, those are two different values...